web/themes/custom/asulib_barrio/templates/views/views-view-unformatted.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for row in rows %}
+  {%
+    set row_classes = [
+      default_row_class ? 'views-row',
+      'css-pick'
+    ]
+  %}
+  
+  <div{{ row.attributes.addClass(row_classes) }}>
```

Comment:
what is this template being used for?

Reply:
I wasn't supposed to add this to this branch, but something somewhere along the line reported that the template file was missing. I think it was even a watchdog message, but I'll push again after removing it.